dactyl-config.yml

```
@@ -3113,6 +3113,18 @@ pages:
         targets:
             - ja
 
+    -   name: Clio Server Methods
+        html: clio-methods.html
+        parent: public-api-methods.html
+        blurb: Use these methods to retrieve information using Clio server APIs.
+        template: pagetype-category.html.jinja
+        targets:
+            - en
+
+    -   md: references/http-websocket-apis/public-api-methods/clio-methods/nft_info.md
+        targets:
+            - en
```

By convention, untranslated pages should be included in the Japanese target if there's no translated version. This might fix the link checking error you're seeing with the 404 page.

```suggestion
        targets:
            - en
            - ja

    -   md: references/http-websocket-apis/public-api-methods/clio-methods/nft_info.md
        targets:
            - en
            - ja
```